src/home/home.scss

```diff
@@ -175,15 +175,16 @@
       column-gap: 10%;
       font-family: Inter;
 
-      // .pic {
-      //   img {
-      //     width: 450px;
-      //     height: 450px;
-      //   }
-      // }
+      .pic {
+        img {
+            border-radius: 6px;
```

Comment:
I can't approve this change.  It is too significant a difference from what came from the designers.  Feel free to create a card on this for Renc to consider (show him a screenshot of the curvy version).  But please remove this change from this PR.